itest/run.sh

```
@@ -0,0 +1,88 @@
+
+BITCOIND_HOST_PORT="${BITCOIND_HOST_PORT:-18443}"
+LND_HOST_PORT="${LND_HOST_PORT:-8081}"
+TAPD_HOST_PORT="${TAPD_HOST_PORT:-8289}"
+export BITCOIND_HOST_PORT LND_HOST_PORT TAPD_HOST_PORT
+
+bc() { $COMPOSE exec -T bitcoind bitcoin-cli -regtest -rpcuser=polaruser -rpcpassword=polarpass "$@"; }
+
+cleanup() {
+  $COMPOSE logs --no-color >"$ITEST_DIR/compose.log" 2>&1 || true
+  $COMPOSE down -v --remove-orphans || true
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)